homeassistant/components/zwave/__init__.py

```diff
@@ -221,22 +221,12 @@ def setup(hass, config):
     descriptions = conf_util.load_yaml_config_file(
         os.path.join(os.path.dirname(__file__), 'services.yaml'))
 
-    try:
-        import libopenzwave
-    except ImportError:
-        _LOGGER.error("You are missing required dependency Python Open "
-                      "Z-Wave. Please follow instructions at: "
-                      "https://home-assistant.io/components/zwave/")
-        return False
     from pydispatch import dispatcher
     # pylint: disable=import-error
     from openzwave.option import ZWaveOption
     from openzwave.network import ZWaveNetwork
     from openzwave.group import ZWaveGroup
 
-    default_zwave_config_path = os.path.join(os.path.dirname(
```

Comment:
No, it does, it's just in a different place. The behavior when config_path is None is now basically an improved version of what we were doing (it checks the new location, along with a number of other places where it has been historically). There's no longer a need for us to get clever with the default path.